src/mastodon/mastodon.rs

```diff
@@ -1504,6 +1504,20 @@ impl megalodon::Megalodon for Mastodon {
         ))
     }
 
+    async fn get_status_source(&self, id: String) -> Result<Response<MegalodonEntities::StatusSource>, Error> {
+        let res = self
+            .client
+            .get::<MegalodonEntities::StatusSource>(format!("/api/v1/statuses/{}/source", id).as_str(), None)
```

Comment:
Please do not use MegalodonEntities here. Please define `StatusSource` in mastodon entities, and use it here. And then convert it to `MegalodonEntities::StatusSource` at L1514.